### PR TITLE
ENYO-6200: Req: ProgressBar: add prop to fix direction of ProgressTooltip

### PR DESCRIPTION
**Pull request by Sangwook-Lee**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [ ] Documentation was added or is not needed

* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
That requirement is adding prop to fix direction of ProgressToolip when propertion is greater than 0.5.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Added a new prop `noTooltipFlip`. (The prop name can be changed after review.)

### Links
[//]: # (Related issues, references)
ENYO-6200

### CODE REVIEW

File: packages/moonstone/ProgressBar/ProgressBarTooltip.js [hunk 1]
```diff
@@ -44,6 +44,15 @@ const ProgressBarTooltipBase = kind({
 	name: 'ProgressBarTooltip',
 
 	propTypes: /** @lends moonstone/ProgressBar.ProgressBarTooltip.prototype */{
+		/**
+		 * Disables tooltip reversal if proportion is greater than 0.5.
+		 *
+		 * @type {Boolean}
+		 * @default true
```
Comment (webOS101) [inline packages/moonstone/ProgressBar/ProgressBarTooltip.js]: This should be `false`, I think.

**Comment by webOS101** [comment]
If only for horizontal, let's spell that out clearly in the help text.

File: packages/moonstone/ProgressBar/ProgressBarTooltip.js (hunk 1 above)
Comment (Sangwook-Lee) [reply packages/moonstone/ProgressBar/ProgressBarTooltip.js]: fixed. thanks.

**Comment by Sangwook-Lee** [comment]
@webOS101 , @sugardave .
As Roy mentioned, it doesn't flip anyhow. I've added help text to notify this prop is for horizontal tooltip only.

**Comment by ryanjduffy** [comment]
Closing in favor of #2565

**Pull request closed without merging**